### DISCUSSION TITLE
Project Workflows `undefinedvalue`

### DISCUSSION THREAD

**Discussion by leighton-tidwell** [community/community#32961] [Projects and Issues] [answered]
I'm trying to set the workflow actions for item closed, but I just see this:
![image](https://user-images.githubusercontent.com/5083652/190015727-8471586c-eaf8-444c-8466-fb1c80423228.png)

**Comment by 0746106363** [comment]
So what am i supposed to do

kim

**Reply by sharma62** [reply]
Did you try the top right option?

**Comment by JandenMa** [comment]
I met the same issue, it seems bcoz you deleted a status that the flow is using.

**Comment by 2percentsilk** [comment, marked as answer]
Hi folks, this was a bug that should be fixed now. Please let me know if that's not the case for you.

**Reply by JandenMa** [reply]
seems fixed. thank you 🥇

**Reply by JandenMa** [reply]
BTW, shall we separate the issue and pull request flows into 2 different flows? right now in the default flows, we only can pick one of PR and issue, or both as the condition, but sometimes we need to separate them. Maybe it is a specific requirement from me only, hahaha. Thanks in advance.

**Reply by gsoumakian** [reply]
Yes on the classic projects, you could move a closed PR to one column and a closed issue to a different column.  This is not possible as I can tell right now.
